fluidkeys/keyrotate.go

```diff
@@ -206,6 +222,7 @@ func printImportBackIntoGnupg(keys []*pgpkey.PgpKey) {
 
 	for _, key := range keys {
 		fmt.Printf("     [ ] Import %s back into GnuPG\n", displayName(key))
+		fmt.Printf("     [ ] Backup %s\n", displayName(key))
```

Comment:
I think it would be easier on the eye if it showed all the "import back into..." followed by "backup"... rather than alternating 
![image](https://user-images.githubusercontent.com/254290/46822144-92f87880-cd82-11e8-99c9-755ede31c3e5.png)